forms-flow-web/Dockerfile.prod

```diff
@@ -0,0 +1,32 @@
+# Used as a reference for any custom logic to be done.
+# if the application is built from a source repo other than aot , use the custom copying.
+# steps : checkout the aot source code and use the copy logic.
+# -----------------------------------------------------------
+# base image
+FROM artifacts.developer.gov.bc.ca/docker-remote/node:14.18.3-alpine as build-stage
```

Comment:
Same thing with here with node version. If no issue with dependencies, v16 is more preferred.